dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -87,13 +89,26 @@ def authenticate
       user = Queries::Lti.get_user(decoded_jwt)
       target_link_uri = decoded_jwt[:'https://purl.imsglobal.org/spec/lti/claim/target_link_uri']
 
+      launch_context = decoded_jwt[Policies::Lti::LTI_CONTEXT_CLAIM]
+      nrps_url = decoded_jwt[Policies::Lti::LTI_NRPS_CLAIM][:context_memberships_url]
+      resource_link_id = decoded_jwt[Policies::Lti::LTI_RESOURCE_LINK_CLAIM][:id]
+      deployment_id = decoded_jwt[Policies::Lti::LTI_DEPLOYMENT_ID_CLAIM]
+      deployment = Queries::Lti.get_deployment(integration.id, deployment_id)
+      redirect_params = {
+        lti_integration_id: integration.id,
+        deployment_id: deployment.id,
+        context_id: launch_context[:id],
+        rlid: resource_link_id,
+        nrps_url: nrps_url,
+      }
+
       if user
         sign_in user
-        redirect_to target_link_uri
+        redirect_to target_link_uri + '?' + redirect_params.to_query
```

Comment:
Should we use string interpolation here? From what I understand it's generally preferred in most cases.